src/tests/tracing/eventpipe/processinfo/processinfo.cs

```diff
@@ -224,14 +224,16 @@ public static void TestEntryPoint()
                 Architecture.X64 => "x64",
                 Architecture.Arm => "arm32",
                 Architecture.Arm64 => "arm64",
+                Architecture.RiscV64 => "riscv64",
```

Comment:
```
string expectedArchValue = RuntimeInformation.ProcessArchitecture switch
{
    Architecture.Arm => "arm32",
    // All other architectures match the enum member name
    _ => RuntimeInformation.ProcessArchitecture.ToString().ToLowerInvariant()
}
```
Can we make this more succinct and reduce need for updates with every new architecture?

Reply:
Thanks, I've updated PR